MapboxDirections/MBRouteOptions.swift

```diff
@@ -332,7 +335,11 @@ open class RouteOptions: NSObject, NSSecureCoding {
         }
         
         if let locale = locale, let languageCode = (locale as NSLocale?)?.object(forKey: .languageCode) as? String {
-            params.append(URLQueryItem(name: "language", value: languageCode))
+            if supportedLocales.contains(languageCode) {
```

Comment:
This will fail for a Locale with the identifier `zh-Hans`, since its `languageCode` will be just `zh`.